components/PlantCard/PlantCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Plant } from '@/types/schema';
+import styles from './PlantCardStyles.module.css';
+
+export default function PlantCard(plantObj: Plant) {
```

Comment:
I think this is what's causing the syntax error. The way you're passing in the plantObj now makes it seem like you're passing the individual fields in plant as props, which is why you have to use the spread operator in `view-plants` (<PlantCard {...plant}><PlantCard/>). 

However, if we want to pass the entire Plant object, we should define PlantCard like this: 
`PlantCard({ plant }: { plant: Plant })`
Then, in `view-plants`, you can use the component like this <PlantCard plantObj={plant}/>

Reply:
also, nit, but i think we can rename the `plantObj` prop to just `plant`